frontend/src/components/Button.tsx

```diff
@@ -19,8 +19,8 @@ const StyledButton = styled.button<{
   box-shadow: ${(props) => (props.state === 'primary' ? `0px` : `2px 1000px 1px #fff inset`)};
   color: ${(props) => (props.state === 'primary' ? `white` : `#58B994`)};
   padding: ${(props) => (props.size === 'small' ? `7px 14px` : `16px 30px`)};
-  line-height: 1px;
-  white-space: nowrap;
+  white-space: normal;
+  cursor: pointer;
```

Comment:
```suggestion
  line-height: 1px;
  white-space: normal;
  cursor: pointer;
```

Reply:
don't remove the `line-height: 1px;` line because it is changing how the button looks like elsewhere in the app (remember this button is a reusable component so do take note as well!)

![image](https://user-images.githubusercontent.com/75428027/200248334-bf686810-ab99-4d64-8603-59468b0e8788.png)

Reply:
I added back the line height code, and the button on your front page still looks good though. so you just double check and add back the line height ya